### PR TITLE
changes to fix incorrect equation with wrong operator symbols

### PR DESCRIPTION
**Pull request by sachingrover211**

### CODE REVIEW

File: www/js/equation.js [hunk 1]
```diff
@@ -135,8 +139,7 @@ define([
 					connections: connections,
 					dynamicList: dynamicList,
 					equation: expressions[0].toString() + " " + this.equalto + " " + expressions[1].toString(),
-					success: !isError,
-					priorError: isError
+					error: isError,
```
Comment (jonwetzel) [inline www/js/equation.js]: Remove extra ","

File: www/js/controller.js [hunk 2]
```diff
@@ -1000,7 +1000,7 @@ define([
 					}));
 				}
 
-				if(parseObject.priorError){ //priorError specifically indicates if there is an error where in a non dynamic node/variable is used inside prior function
+				if(parseObject.error){ //error specifically indicates if there is an error where in a non dynamic node/variable is used inside prior function
```
Comment (jonwetzel) [inline www/js/controller.js]: This comment is no longer true, I think.   Now it indicates other types of errors too.  (That's one of the reasons you renamed it I think.)  Please update the comment as needed.

Comment (sachingrover211) [reply www/js/controller.js]: This comment is true, I updaed it for different reason. Earlier I am using success and priorError for the same variable. So I gave it one generic name. It is still the marked for prior error only.

**Comment by sachingrover211** [comment]
@jonwetzel I have made the appropriate changes. Kindly let me know if there is anything else that I need to do for this. Thanks

**Comment by jonwetzel** [comment]
@sachingrover211 Merge conflict detected.

**Comment by sachingrover211** [comment]
@jonwetzel Merge conflict fixed.

**Pull request merged**